controllers/topics_controller.js

```diff
@@ -0,0 +1,16 @@
+const { request, response } = require("express");
+const db = require("../db/connection");
+const { allTopics } = require("../models/topics_model");
+
+const getTopics = (request, response, next) => {
+  allTopics(request.query)
```

Comment:
We don't need to worry about queries for this ticket (spoiler- this is gonna be ticket 11!)